src/EasyAbp.BookingService.Application/EasyAbp/BookingService/Assets/AssetAppService.cs

```diff
@@ -75,6 +76,7 @@ protected override async Task MapToEntityAsync(CreateUpdateAssetDto updateInput,
             updateInput.Volume,
             updateInput.Priority,
             ObjectMapper.Map<TimeInAdvanceDto, TimeInAdvance>(updateInput.TimeInAdvance),
-            updateInput.Disabled);
+            updateInput.Disabled,
+            updateInput.ExtraProperties);
     }
```

Comment:
Should use `createInput.MapExtraPropertiesTo(asset)` after the entity is created.